website/source/docs/builders/azure.html.md

```diff
@@ -100,13 +102,130 @@ Here is a basic example for Azure.
     "capture_container_name": "images",
     "capture_name_prefix": "packer",
 
+    "os_type": "Linux",
     "image_publisher": "Canonical",
     "image_offer": "UbuntuServer",
-    "image_sku": "14.04.3-LTS",
+    "image_sku": "14.04.4-LTS",
 
     "location": "West US",
     "vm_size": "Standard_A2"
 }
 ```
 
-See the [examples/azure](https://github.com/mitchellh/packer/tree/master/examples/azure) folder in the packer project for more examples.
+## Device Login
```

Comment:
The first section mentions authorization:

    Learn how to [authorize access to ARM](/docs/builders/azure-setup.html).

I think it would be good to put explanations for both types of auth in the same place. If you feel like the other page is cluttered or better-suited to other things, feel free to create a new page (e.g. Azure Authorization) and add a link to the sidebar; we can link to it from the appropriate place on both the main configure page and the getting started guide.